conda-forge.yml

```diff
@@ -4,7 +4,7 @@ test_on_native_only: true
 azure:
   settings_win:
     pool:
-      vmImage: windows-2019
+      vmImage: windows-2022
```

Comment:
:point_up: `windows-2022` is now the default (https://github.com/conda-forge/conda-smithy/pull/1720) so this setting can be ommitted as of early 2023.

This comment is for context for people who follow a link from the docs to here :)